### ISSUE TITLE
seg loss: "BCEseg = nn.BCEWithLogitsLoss"

### ISSUE THREAD

**Issue by crisz94** [hustvl/YOLOP#117]
thanks for your brilliant work.

I noticed that you use nn.sigmoid for seg head output. In this case, nn.BCELoss should be used for loss computing instead of  nn.BCEWithLogitsLoss.

Actually, nn.functional.cross_entropy is commonly used in seg tasks.

**Comment by dedellzyw** [comment]
yes! I think you are right. This will causes loss_seg to become nan.
